postgres-appliance/tests/test_spilo.sh

```diff
@@ -286,6 +286,9 @@ function test_spilo() {
     create_schema2 "$container" || exit 1
 
     run_test test_pg_upgrade_to_13_check_failed "$container"  # pg_upgrade --check complains about OID
+    drop_table_with_oids "$container"
+    wait_backup "$container"
+    wait_zero_lag "$container"
```

Comment:
Well, I actually meant running `wait_backup` and `wait_zero_lag` right before `start_clone_with_wale_upgrade_container`. Like it used to be [here](https://github.com/zalando/spilo/commit/588b7f7c7efc50c26f1ad6c9a1e7898170bb417b#diff-b6bee1e27ff237ba64f478e8291c6ee044267dd68c8d618b90bca01f8b4b4bd8L343-L347)